package.json

```diff
@@ -31,14 +31,19 @@
 			"email": "jo@3yourmind.com",
 			"name": "James O'Shea",
 			"url": "https://github.com/jamesoshea"
+		},
+		{
+			"email": "tquinonero@gmail.com",
+			"name": "Tom Quinonero",
+			"url": "https://github.com/tomquinonero"
 		}
 	],
 	"dependencies": {
-		"@3yourmind/yoco": "^1.1.0",
-		"color": "^3.1.0",
```

Comment:
(1) I don't know how stable these other packages are, but IMHO the ones under our control should be ~. cc @amrdraz

Reply:
I'm with @mes3yd I don't particularly trust, all packages, I've added a suggestion to change back the ones think may not respect minor version